src/generators/minecraft-character-allay/MinecraftAllayCharacterGenerator.bs.js

```diff
@@ -0,0 +1,2220 @@
+// Generated by ReScript, PLEASE EDIT WITH CARE
```

Comment:
You can remove this file. We only keep the `.res` files, and the `.bs.js` files are generated when we need them.

Normally doing a "git commit" command will prevent these from being included. How are you doing your commits? Maybe I can suggest a way that might be a bit easier and avoid this happening.